ts/features/pn/navigation/routes.ts

```diff
@@ -1,7 +1,7 @@
 const PN_ROUTES = {
-  MAIN: "PN_ROUTES_MAIN",
```

Comment:
This is a breaking change, since there are FAQ mapped on these routes

Reply:
Fixed https://github.com/pagopa/io-app/pull/5091/commits/ccc9449f44fbb22c964db4e922f2dfb187ff3a71